src/test/java/io/confluent/examples/streams/FanoutLambdaIntegrationTest.java

```diff
@@ -88,56 +69,56 @@ public void shouldFanoutTheInput() throws Exception {
 
     final Properties streamsConfiguration = new Properties();
     streamsConfiguration.put(StreamsConfig.APPLICATION_ID_CONFIG, "fanout-lambda-integration-test");
-    streamsConfiguration.put(StreamsConfig.BOOTSTRAP_SERVERS_CONFIG, CLUSTER.bootstrapServers());
+    streamsConfiguration.put(StreamsConfig.BOOTSTRAP_SERVERS_CONFIG, "dummy config");
     streamsConfiguration.put(StreamsConfig.DEFAULT_VALUE_SERDE_CLASS_CONFIG, Serdes.String().getClass().getName());
-    streamsConfiguration.put(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "earliest");
 
+    final String inputTopicA = "A";
+    final String outputTopicB = "B";
+    final String outputTopicC = "C";
     final KStream<byte[], String> stream1 = builder.stream(inputTopicA);
-    final KStream<byte[], String> stream2 = stream1.mapValues((v -> v.toUpperCase()));
-    final KStream<byte[], String> stream3 = stream1.mapValues(v -> v.toLowerCase());
+    final KStream<byte[], String> stream2 = stream1.mapValues(((ValueMapper<String, String>) String::toUpperCase));
```

Comment:
nit: I understand the cast to disambiguate from`ValueMapperWithKey` but is casting any better than `v -> v.toUpperCase()`?

Reply:
It's debatable... I just took the suggestion from IDEA. Happy to roll it back.

Reply:
just leave it.

Reply:
too late :)